.github/workflows/apply.yml

```diff
@@ -117,16 +144,19 @@ jobs:
           cat <<- EOF >> terraform.tfvars.json
             ${{ inputs.tf_var_json }}
           EOF
-          output=$(terraform apply --auto-approve --input=false -no-color)
-          echo $output
+          terraform apply --auto-approve --input=false -no-color 2>&1 | tee tf_output.txt
```

Comment:
Does tee create tf_output.txt if it doesn't exist?

Reply:
Yes... I might need to clean up that file afterwards, though